Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Toggle.Switch/Classes/GraphicsExtensionMethods.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Drawing;
+
+namespace Krypton.Toolkit.Suite.Extended.Toggle.Switch
+{
+    public static class GraphicsExtensionMethods
+    {
+        public static Color ToGrayScale(this Color originalColour)
```

Comment:
This should not be necessary, as it should be using the Krypton them style for disabled.

Reply:
I think that belonged to: https://www.codeproject.com/Articles/1029499/ToggleSwitch-Winforms-Control, but I haven't converted it to fully use Krypton yet.

Reply:
Need to convert at a later date